src/Commands/BuildToolsCommand.php

```diff
@@ -658,9 +659,13 @@ protected function expandSourceAliases($source)
      */
     protected function autodetectUpstream($siteDir)
     {
-        return 'Empty Upstream';
-        // or 'Drupal 7' or 'WordPress'
-        // return 'Drupal 8';
+        if (file_exists($siteDir . '/wp-cli.yml')) {
+            return 'WordPress';
```

Comment:
I created a new empty upstream w/ the framework set to 'wordpress' named 'Empty WordPress'